biogeochem/EDPhysiologyMod.F90

```diff
@@ -2849,6 +2851,8 @@ subroutine CWDInput( currentSite, currentPatch, litt, bc_in)
     integer  :: numlevsoil        ! Actual number of soil layers
 
     real(r8) :: SF_val_CWD_frac_adj(4) !SF_val_CWD_frac adjusted based on cohort dbh
+    real(r8) :: leaf_herbivory
+    real(r8) :: herbivory_element_use_efficiency
     !----------------------------------------------------------------------
```

Comment:
Maybe give a short definition of these two? leaf_herbivory and herbivory_element_use_efficiency